opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/wiremock/mocks/WiremockAccountInformationRequest.java

```diff
@@ -97,8 +98,8 @@ public SELF user_anton_brueckner_provided_initial_parameters_but_without_psu_id_
 
         ExtractableResponse<Response> response = RestAssured
                  .given()
+                     .header(X_REQUEST_ID, DEFAULT_X_REQUEST_ID)
```

Comment:
Consent API calls should use generated request IDs (like it was before)

Reply:
random UUID is generated